### ISSUE TITLE
cant get gesture data

### ISSUE THREAD

**Issue by JoaoMenezes** [derekrazo/o.io.leap#2]
Cant get gesture data from [o.io.leap] outlet.

They don't appear in the o.message connected to o.io.leap outlet, nor printed with o.print. 

Also tried to route them through o.route, without any success.  

on:
MacBook Pro . 15-inch, Mid 2012
OS X 10.9.1 (13B42)
2,6 GHz Intel Core i7 / 8 GB 1600 MHz DDR3 / Intel HD Graphics 4000 1024 MB

**Comment by derekrazo** [comment]
I have gone back and checked my system. Looks like the gestures are working for me. I notice you are on mavericks. I will try to get a mavericks machine to test on my end!

**Comment by JoaoMenezes** [comment]
Yes, i forgot the mavericks fact. However, please note that i'm able to
recognize gestures through a conventional sampleListener that i'm running
on xcode.

Thank you so much,
JM

João Menezes
www.cargocollective.com/joaomenezes
http://joaomenezesdev.tumblr.com/
https://github.com/joaomenezes

**Comment by JoaoMenezes** [comment]
Yes, i forgot the mavericks fact. However, please note that i'm able to recognize gestures through a conventional sampleListener that i'm running on xcode.

**Comment by derekrazo** [comment]
Hm. To be honest I'm pretty swamped right now. I can try to get to it in
the next week, or I can just give you the odot library and you can try to
build yourself.

What do you think?
